doc/input-output-reference/src/overview/group-zone-forced-air-units.tex

```diff
@@ -4560,17 +4588,17 @@ \subsubsection{Outputs}
+\paragraph{Zone Hybrid Unitary HVAC Supply Air Temperature [\si{\degreeCelsius}]}
+This output reports the supply air temperature. For each timestep the value is calculated as a supply air mass weighted average of the supply air temperature for each of the settings selected for the time step. For example, if the system operates for half of the timestep with supply air mass flow rate of 1 kg/s and supply air temperature of 16\si{\degreeCelsius}, and for half of the timestep at 2 kg/s and 10\si{\degreeCelsius}, the supply air temperature calculated for the time step would be 12 \si{\degreeCelsius}. The output is the average over the reporting period.
 
-\paragraph{Zone Hybrid Unitary HVAC Return Air Temperature [C]}
+\paragraph{Zone Hybrid Unitary HVAC Return Air Temperature [\si{\degreeCelsius}]}
 This output reports the return air temperature. The return air temperature is inherited from the associated zone outlet node in each timestep. The output is the average over the reporting period.
 
-\paragraph{Zone Hybrid Unitary HVAC Outdoor Air Temperature [C]}
+\paragraph{Zone Hybrid Unitary HVAC Outdoor Air Temperature [\si{\degreeCelsius}]}
 This output reports the outdoor air temperature.  The outdoor air temperature is inherited from the associated outdoor air node in each timestep.  The output is the average over the reporting period.
 
 \paragraph{Zone Hybrid Unitary HVAC Supply Air Humidity Ratio [kgWater/kgDryAir]}
-This output reports the supply air humidity ratio. For each timestep the value is calculated as a supply air mass weighted average of the supply air humidity ratio for each of the settings selected for the time step. For example, if the system operates for half of the timestep with supply air mass flow rate of 1 kg/s and supply air humidity ratio of 0.016 kg/kg, and for half of the timestep at 2 kg/s and 0.010 kg/kg, the supply air humidity ratio calculated for the time step would be 0.012 °C. The output is the average over the reporting period.
+This output reports the supply air humidity ratio. For each timestep the value is calculated as a supply air mass weighted average of the supply air humidity ratio for each of the settings selected for the time step. For example, if the system operates for half of the timestep with supply air mass flow rate of 1 kg/s and supply air humidity ratio of 0.016 kg/kg, and for half of the timestep at 2 kg/s and 0.010 kg/kg, the supply air humidity ratio calculated for the time step would be 0.012\si{\degreeCelsius}. The output is the average over the reporting period.
 
```

Comment:
There is an error here in the units (i.e., 0.012 C). The existing units were converted correctly, however, the units should be kgWater/kgDryAir. Also, the previous sentence uses kg/kg, these units could also be updated to be consistent. This is the only issue I found with the doc changes.

Reply:
@jmythms there is a correction needed in the doc. This gives you the opportunity to Clang format the hh as needed (and cc? but I didn't see anything there). Give me a little time for more review before you make a commit.

Reply:
That was a stupid mistake on my part. I will make the changes, and add the reformatted "src/EnergyPlus/PurchasedAirManager.hh" file in with that commit.

Reply:
Fixed this:
![image](https://user-images.githubusercontent.com/45446967/107706095-05dccb80-6c7d-11eb-93e9-408c9d98d0b8.png)